person.rb

```diff
@@ -2,10 +2,11 @@
 require './nameable'
 require './capitalize_decorator'
 require './trimmer_decorator'
+require './rental'
 
 class Person < Nameable
-  attr_accessor :name, :age
-  attr_reader :id, :rentals
+  attr_accessor :name, :age, :rentals
```

Comment:
**[OPTIONAL BUT REQUIRED]**

- [ ] Hey, though I marked this optional because it wasn't pointed out by the previous reviewer, kindly note that it is a required change you should implement. :pray: 
   - Please, you should add the `:books` to the attr_accessor also in order to be able to read and write it as remember the intermediate intersection between a person and a book is the rentals, so one needs to be able to access the book.

Reply:
hi @zieeco,
please i don't really get what you mean here, because there is no books instance variable inside class Person.
please can you explain me more ?